mlir/utils/jenkins/Jenkinsfile

```diff
@@ -522,7 +522,7 @@ pipeline {
                             script {
                                 build_fixedE2ETests("${CODEPATH}")
                                 preMergeCheck("${CODEPATH}")
-                                timeout(time: 60, activity: true, unit: 'MINUTES') {
+                                timeout(time: 120, activity: true, unit: 'MINUTES') {
```

Comment:
This timeout is on the activity not on the how much time is spent on running the tests. 
So if it doesn't print anything on the logs for 60 minutes that is when it times out. 

Therefore i think we don't need  increase the timeout limit. 

You can look up `activity: true` flag meaning.